sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/backend/rust/RustSqlRuntimeLibrary.java

```diff
@@ -56,6 +56,7 @@ protected RustSqlRuntimeLibrary() {
         this.arithmeticFunctions.put("lte", DBSPOpcode.LTE);
         this.arithmeticFunctions.put("gte", DBSPOpcode.GTE);
         this.arithmeticFunctions.put("plus", DBSPOpcode.ADD);
+        this.arithmeticFunctions.put("safe_add", DBSPOpcode.SAFE_ADD);
```

Comment:
because safe_add is a very strange function and does not really behave like add, I would actually refrain from adding a new operator  and I would treat it like a function.